releasenotes/notes/http-writer-reset-090a35e52645e36e.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Internal: Ensure the http connection is correctly reset when ``Response.from_http_response`` raises an exception.
```

Comment:
Can we update the scope to include all impacted products (ex: tracing,civis)

Reply:
Done, thanks!